lex/Makefile

```diff
@@ -16,7 +16,11 @@
 # The lexer generator
 
 ROOTDIR = ..
+# NOTE: it is important that OCAMLLEX is defined *before* Makefile.common
+# gets included, so that its definition here takes precedence
+# over the one there.
 
```

Comment:
```suggestion

# NOTE: it is important that OCAMLLEX is defined *before* Makefile.common
# gets included, so that its definition here takes precedence
# over the one there.
```
(whitespace adjustment)